Documentation/Guides/Changelog.md

```diff
@@ -1,3 +1,8 @@
+## 5.1.1
+
+- [BUGFIX] Fixes a critical bug that would cause apps to freeze when calling `trackEvent` in UIApplicationDelegate callbacks. [#492](https://github.com/bitstadium/HockeySDK-iOS/pull/493)
+- [BUGFIX] Fix a critical bug in the crashonly variant of the SDK. [#49](https://github.com/bitstadium/HockeySDK-iOS/pull/494)
+
```

Comment:
it's really odd that this is displayed as a change while it's already in develop.